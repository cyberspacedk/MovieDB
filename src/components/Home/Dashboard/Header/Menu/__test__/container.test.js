import React from 'react';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import MenuContainer from '../container';

describe('Component: Home', () => {
  const store = configureStore()({
    user: {
      username: 'User-1',
      sessionId: 'jchdi3bmc99aj',
    },
  });
  const container = shallow(<MenuContainer store={store} />);

  it('Match its snapshot', () => {
    expect(container).toMatchSnapshot();
  });
});
