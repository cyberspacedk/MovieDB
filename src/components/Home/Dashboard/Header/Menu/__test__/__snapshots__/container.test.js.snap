// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Home Match its snapshot 1`] = `
<UserMenu
  handleLogout={[Function]}
  username="User-1"
/>
`;
